docs/pages/database-access/guides/sql-server-ad.mdx

```diff
@@ -388,7 +388,9 @@ If your `tsh db connect` error includes the following text, the certificate used
 Error message: TLS Handshake failed: x509: certificate signed by unknown authority
 ```
 
-Add the Certificate Authority (CA) `ca_cert_file` into the `tls` section so Teleport can validate the certificate. 
+To solve this, you can add the CA configuration to the database like the
+following:
```

Comment:
Same comment as the one I left on this text in the `sql-server-ad-pkinit.mdx` guide